projects/ngx-drag-to-select/src/lib/select-container.component.ts

```diff
@@ -416,7 +417,7 @@ export class SelectContainerComponent implements AfterViewInit, OnDestroy {
       const itemRect = item.getBoundingClientRect();
       const withinBoundingBox = inBoundingBox(mousePoint, itemRect);
 
-      if (this.shortcuts.extendedSelectionShortcut(event)) {
+      if (this.shortcuts.extendedSelectionShortcut(event) || this.additive) {
```

Comment:
What about extracting this condition to a function?

Reply:
What will you call it?

And this condition?
```
 this.shortcuts.addToSelection(event) || this.additive
```

Reply:
I think it's fine to keep this inline as this is just a simple flag and not a complex condition. What do you think?

Reply:
I think it can be easily read as it is. 
If we extract this condition to a method we will have to move others like the one in my comment to keep the code consistent.

Reply:
👍